x-pack/plugins/uptime/public/components/monitor/ping_list/columns/ping_timestamp/ping_timestamp.tsx

```diff
@@ -110,12 +111,14 @@ export const PingTimestamp = ({ timestamp, ping }: Props) => {
           isPending={status === FETCH_STATUS.PENDING}
         />
       )}
-      <NavButtons
-        maxSteps={data?.maxSteps}
-        setIsImagePopoverOpen={setIsImagePopoverOpen}
-        setStepNumber={setStepNumber}
-        stepNumber={stepNumber}
-      />
+      {showNavBtn && (
```

Comment:
I thought we decided to remove the navigation element on the ping timestamp and just show one image?

Reply:
i wasn't aware of that conversation, i believe that must be a separate issue.